tiatoolbox/models/dataset/info.py

```diff
@@ -70,7 +73,8 @@ class name as a string.
     def __init__(
         self,
         save_dir_path=None,
-    ):
+    ) -> None:
+        """Initializes :class:`KatherPatchDataset`."""
```

Comment:
```suggestion
        """Initialize :class:`KatherPatchDataset`."""
```

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/625/commits/c2e1cd4ccfd6aaeb32664f23d4f6634cddb90090